src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.sb__form {
+  margin: 20px 0;
+}
+
+.search__bar {
+  width: 100%;
+
+  color: #3d4e61;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  outline: none;
+}
+
+.search__bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search__bar-big {
+  padding: 25px 62px 23px;
+
+  font-size: 16px;
+  line-height: 22px;
+
+  background: url(images/Search.svg) no-repeat 26px 25px;
+}
+
+.search__bar-big.search__bar-big:focus {
+  font-weight: bold;
+  box-shadow: none;
+
+  background: url(images/Search.svg) no-repeat 26px 25px,
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search__bar-small {
+  padding: 12px 33px 10px;
+
+  width: 50%;
```

Comment:
Search bar must have width 100%, not 50%

Reply:
I agree, tnx  

I've just corrected in commit 'version 2.0'